src/script/backport-create-issue

```diff
@@ -257,6 +257,8 @@ def update_relations(r, issue, dry_run):
             other = r.issue.create(**create_args)
         except ValidationError as e:
             # try without an assignee
+            logging.debug("Creating the issue failed: {e}")
```

Comment:
Nit: This is inside of a loop, so I'd make more precise:
```suggestion
            logging.debug("Creating issue for {release} failed: {e}")
```

Reply:
ack, look again

Reply:
Looks much better to me!